cypress/elements/fileMenu/index.js

```diff
@@ -1,57 +1,54 @@
-/*  FIXME:
-    Most of the element selection in this file is quite poor because 
-    of the FileMenu being old and still uses MUI.
-    This can be refactored to use proper data-test selectors once the new
-    FileMenu component is in place.
-*/
-
 import { clickMenuBarFileButton } from '../menuBar'
 
-const deleteModalEl = '*[class^="MuiDialog-container"]'
-const fileMenuItemEl = '*[role="menuitem"]'
+const deleteModalEl = 'file-menu-delete-modal'
+const fileMenuItemEl = 'file-menu-container'
```

Comment:
If the content is `file-menu-container`, then this should probably be renamed to `fileMenuContainerEl`

Reply:
Done.